scadalts-ui/tests/utils/testing-utils.js

```diff
@@ -82,6 +82,7 @@ export function prepareShallowMountWrapper(
 		vuetify,
 		store,
 		i18n,
+		router,
```

Comment:
What is the justification for this?

Reply:
If the router is not added to tests, SynopticPanel testing don't work because this.$router is undefined.

https://github.com/SCADA-LTS/Scada-LTS/actions/runs/10505804935/job/29104184414

I dont get how testing was working before upgrading.